NEWS.md

```diff
@@ -2,6 +2,10 @@
 
 * POST /item-storage/items error message on non-UUID statistical code ID (MODINVSTOR-755)
 
+## 24.0.1 2022-07-18
+
+* Improve populating shelfKey from callNumber ([MODINVSTOR-932](https://issues.folio.org/browse/MODINVSTOR-932))
```

Comment:
This pull request includes MODINVSTOR-755 into the 24.0.1 release. This should be reflected in the NEWS.md file by moving the MODINVSTOR-755 entry from 25.0.0 to 24.0.1.